test/instrumentation/stability-utils.sh

```diff
@@ -29,37 +29,46 @@ stability_check_setup() {
   kube::golang::setup_env
 }
 
-find_files_to_check() {
-  find . -not \( \
-      \( \
-        -wholename './output' \
-        -o -wholename './_output' \
-        -o -wholename './_gopath' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename '*/third_party/*' \
-        -o -wholename '*/vendor/*' \
-        -o -wholename '*/hack/*' \
-        -o -wholename '*_test.go' \
-        -o -wholename './test/instrumentation/*.go' \
-        \) -prune \
-    \) \
-    \( -wholename '**/*.go' \
-  \)
+function find_files_to_check() {
+    # Similar to find but faster and easier to understand.  We want to include
+    # modified and untracked files because this might be running against code
+    # which is not tracked by git yet.
+    git ls-files -cmo --exclude-standard \
+        ':!:vendor/*' \
+        ':!:third_party/*' \
+        ':!:hack/*' \
+        ':!:*/testdata/*' \
+        ':!:*/*_test.go' \
+        ':!:./test/instrumentation' \
+        ':(glob)**/*.go' \
+        "$@"
 }
 
-find_test_files() {
-  find './test/instrumentation' -wholename '**/*.go'
+function find_test_files() {
+  git ls-files -cmo --exclude-standard \
+      ':!:*/*_test.go' \
```

Comment:
Why do you exclude test.go files here? It seems that previously were not excluded

Reply:
yeah, that looks like a mistake!